### ISSUE TITLE
TOS DI is not available on sourceforge 

### ISSUE THREAD

**Issue by nagyrobir** [talend-spatial/talend-spatial#96]
Hi!

It seems that we cannot use talend spatial at all anymore since TOS DI is not available on sourceforge, and we can only download talend 8 from the official website. Any ideas on what is next? is this going to be archived or will there be an upgrade?

**Comment by fxprunayre** [comment]
Indeed, it looks like sourceforge requested to all projects to do some cleanup 
"We need your help removing a lot of your old releases.
Please remove as much as you can of your old releases and unused files within 14 days."
and Talend removed all. I can't find any location to download past releases. There is no update of the spatial module to version 8 planned AFAIK.

**Comment by nagyrobir** [comment]
Is it a budgeting problem in a sense that a crowd funding would be needed or is it a licensing problem, since QLICK owns Talend? or is it not enough users problem?

**Comment by nagyrobir** [comment]
And by the way i would have liked to ask Talend to make version 7.4.1 available somehow, but i dont even know where to ask them.

**Comment by fxprunayre** [comment]
> Any ideas on what is next?

https://www.talend.com/blog/update-on-the-future-of-talend-open-studio/ "we will be discontinuing the open-source product: Talend Open Studio, effective January 31, 2024."
